newIDE/app/src/MainFrame/EditorContainers/HomePage/SectionContainer.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+import * as React from 'react';
+import { Column, Line } from '../../../UI/Grid';
+import { Paper } from '@material-ui/core';
+import { useResponsiveWindowWidth } from '../../../UI/Reponsive/ResponsiveWindowMeasurer';
+import Text from '../../../UI/Text';
+import GDevelopThemeContext from '../../../UI/Theme/ThemeContext';
+
+const styles = {
+  mobileScrollContainer: {
+    padding: 5,
+  },
+  desktopScrollContainer: {
+    paddingTop: 30,
+    paddingLeft: 30,
+    paddingRight: 30,
+    paddingBottom: 10,
+  },
+  titleContainer: {
+    paddingBottom: 20,
+  },
+  scrollContainer: {
+    flex: 1,
+    display: 'flex',
+    overflowY: 'auto',
+  },
+};
+
+type Props = {|
+  children: React.Node,
+  title: React.Node,
+  subtitle?: React.Node,
+|};
+
+export const SectionContainer = ({ children, title, subtitle }: Props) => {
+  const windowWidth = useResponsiveWindowWidth();
+  const GDevelopTheme = React.useContext(GDevelopThemeContext);
+  return (
+    <Column useFullHeight noMargin expand>
+      <Paper
+        style={{
+          ...styles.scrollContainer,
+          borderLeft: `1px solid ${GDevelopTheme.home.separator.color}`,
```

Comment:
Was it necessary to add a color in the theme? This separator color is not present in Material UI's theme?